gm4_calling_bell/data/gm4_calling_bell/functions/tick.mcfunction

```diff
@@ -0,0 +1,4 @@
+#
+execute as @a[scores={gm4_calling_bell_rung=1..}] at @s anchored eyes positioned ^ ^ ^ run function gm4_calling_bell:emerald_check
```

Comment:
The position alignment is not needed here. See my comment in `gm4_calling_bell:emerald_check`.
```suggestion
execute as @a[scores={gm4_calling_bell_rung=1..}] run function gm4_calling_bell:emerald_check
```